qml/pages/PlayerPage.qml

```diff
@@ -33,6 +33,10 @@ Page {
         if (visible) updateCover(qsTr("Now playing"), program.title, program.itemTitle)
     }
 
+    Component.onDestruction: {
+        appWindow.insertStartedProgram({ id: program.id, progress: mediaPlayer.position })
```

Comment:
check if this is called upon closing the app by a swipe and make sure this is called also when pause button is pressed in case the handle isn't called upon closing the app forcefully